src/NzbDrone.Core/Datastore/Converters/CommandConverter.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using Marr.Data.Converters;
+using NzbDrone.Common.Reflection;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Messaging.Commands;
+using NzbDrone.Core.ThingiProvider;
+
+namespace NzbDrone.Core.Datastore.Converters
+{
+    public class CommandConverter : EmbeddedDocumentConverter
+    {
+        public override object FromDB(ConverterContext context)
+        {
+            if (context.DbValue == DBNull.Value)
+            {
+                return NullConfig.Instance;
```

Comment:
What is happening here? `IProviderConfig` isn't convertible to `Command`

copy/past fail?